spec/models/distribution_spec.rb

```diff
@@ -123,6 +123,18 @@
         expect(Distribution.by_partner(partner1.id).size).to eq(1)
       end
     end
+
+    describe "by_location >" do
+      let!(:location_1) { create(:storage_location) }
+      let!(:location_2) { create(:storage_location) }
+
+      it "only returns distributions with given location id" do
+        create(:distribution, storage_location: location_1)
+        create(:distribution, storage_location: location_2)
+
+        expect(Distribution.by_location(location_1.id).size).to eq(1)
+      end
+    end
```

Comment:
```suggestion
    describe "by_location >" do
      let!(:location_1) { create(:storage_location) }
      let!(:location_2) { create(:storage_location) }

      it "only returns distributions with given location id" do
        dist_1 = create(:distribution, storage_location: location_1)
        dist_2 = create(:distribution, storage_location: location_2)

        expect(Distribution.by_location(location_1.id)).to include(dist_1)
        expect(Distribution.by_location(location_1.id)).not_to include(dist_2)
      end
    end
```
Could we update this spec so that it checks the contents of the `Distribution.by_location` method you've added? I think this is could be more robust and tell us the correct value is returned.